css-flexbox/display-flex.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - display: flex</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      .container {
+        display: flex;
+        justify-content: space-between;
+      }
+
+      #card-deck {
+        margin: 8px 8px 0px 8px;
+      }
```

Comment:
Same thing here where this could have been one rule set for `#card-deck` that had all of the rules in it.